ci-operator/config/openshift/microshift/openshift-microshift-main.yaml

```diff
@@ -157,6 +158,14 @@ tests:
     env:
       GOOGLE_COMPUTE_IMAGE_NAME: rhel-9-v20230203
     workflow: openshift-microshift-e2e-reboot-tests
+- as: e2e-greenboot
```

Comment:
Would you like to add `optional: true` until we see if job is stable?

Reply:
Good idea. Let me do this asap.